docs/CONTRIBUTING.md

```diff
@@ -61,8 +61,6 @@ If you're interested in participating in the development of SORMAS, you can use
 
 * [Setting up your local environment](DEVELOPMENT_ENVIRONMENT.md)
 * [Adding license headers](ADDING_LICENSE.md)
-* [How to add a new disease?](GUIDE_ADD_NEW_DISEASE.md)
```

Comment:
I've moved these guides to the Wiki - can you re-add and update the links please?

Reply:
sure thing! :)